ironfish-cli/src/commands/reset.ts

```diff
@@ -54,7 +54,7 @@ export default class Reset extends IronfishCommand {
 
     const message =
       '\nYou are about to destroy your node databases. The following directories and files will be deleted:\n' +
-      `\nAccounts: ${accountDatabasePath}` +
```

Comment:
Can you change `Accounts` in the message?

Reply:
Fixed